src/lib/Libtpp/tpp_em.c

```diff
@@ -1639,7 +1639,7 @@ tpp_mbox_post(tpp_mbox_t *mbox, unsigned int tfd, int cmdval, void *data)
 
 	while (1) {
 		/* send a notification to the thread */
-#ifdef HAVE_EVENTFD_H
+#ifdef HAVE_SYS_EVENTFD_H
```

Comment:
Where is the #endif?

Reply:
At around line 1652

Reply:
Thanks. Not sure how I missed that. Looks fine.